Install.md

```diff
@@ -0,0 +1,86 @@
+# Anomaly Detection Installation
+This document describes how to set up Anomaly Detection for OpenSDS project.
+
+## Pip Installation
+Run command ```which pip``` to check if your environment has pip tool, if not you can refer to the following steps to install it.
+```shell
+$ wget https://bootstrap.pypa.io/get-pip.py
+$ python get-pip.py # Maybe python3 get-pip.py in your environment.
+``` 
+
+## Anomaly Detection Installation
+Install anomaly detection using pip.
+```shell
+pip install git+https://github.com/opensds/anomaly-detection.git
+```
+if you want to install a specified branch, run command like blow:
+```shell
+$ pip install git+https://github.com/opensds/anomaly-detection.git@0.5.3
+```
+## Configuration
+Anomaly detection will get metrics from from Telemetry, before modifying the configuration file, please make sure the telemetry is started up.
+*  Create configuration template file.
```

Comment:
Is there an example config template file?

Reply:
yes, its path is etc/anomaly_detection.conf

Reply:
Can you add example path on line 23?

Reply:
This may make you a little bit confusing. This description only want to tell you copy the template to directory '/etc'. and I have changed the descritption.